### ISSUE TITLE
LV2 synths depend on development package lv2-dev

### ISSUE THREAD

**Issue by Gnlfz007** [muse-sequencer/muse#821]
MusE fails to load any LV2 synth plugin if the devel packets are not installed. I copied a fully functional installation from /usr/local to a second PC and installed all the libraries required by the MusE and the LV2 plugins (checked witd ldd). Muse was fully functional except for the LV2 synths. The LV2 effects were available.
After the installation of liblilv-dev, the LV2 synths also showed up. Of course I'm not sure if this is actually a MusE bug.

**Comment by spamatica** [comment]
As I wrote in the forum post I believe it's liblilv that it depends on. 

I see now that you checked with ldd... might mean that I am wrong. 
In either case I don't think it's a bug with MusE, it is curious though..

**Comment by Gnlfz007** [comment]
I did some more investigation, starting with a cleaner environment (deborphan!).
Result: It is packet "lv2-dev" which makes the LV2 synths show up, not liblilv-dev. I changed the subject of this ticket accordingly. 
Either I mixed up things or liblilv-dev was pulled in by something else.

**Comment by kybos** [comment]
@Gnlfz007 
OT: I noticed by pure incident that you created a bug report in the wiki, which is not the right place, nobody will find or read it there. Please create an issue here. For more general discussions, the forum can be used too.
The wiki is meant for program documentation only. We left it open so all github users can contribute, but it shouldn't be used for anything else.
Thanks.

**Comment by Gnlfz007** [comment]
Hi, I dit not create or modify a Wiki page, unless accidentally. 
Isn't this an issue here in the right place, specifically "issue #821"? Is issue #821 considered a Wiki entry?! Actually I can't understand your message.

**Comment by spamatica** [comment]
Heh, it's midsummer day here, not sure they celebrate that in germany @kybos? ;) Maybe some releated confusion   * innocent look *
I think it was a mistake and the ticket is in the right place.

Very odd too that you managed to reproduce this still.

**Comment by kybos** [comment]
https://github.com/muse-sequencer/muse/wiki/MusE-fails-to-handle-compressed-project-files

**Comment by terminator356** [comment]
Verified, loaded song is blank.
Yes please file a ticket instead of the wiki. Thanks.

**Comment by Gnlfz007** [comment]
Wrap-up: 
- I accidentally created a wiki entry for a completely different issue (compressed project files) and this wiki entry got completely out of sight for me. kybos posted a link to this page and I took some time to get it (see "booze").
- I have created a issue [#831](https://github.com/muse-sequencer/muse/issues/831) for this.
- I have removed the useless wiki page now (wow, why am I allowed to do this? Tanks god I'm not a troll!).

**Btw; the issue "depends on lv2-dev" persists and can be handled here.**
